app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt

```diff
@@ -111,6 +112,7 @@ object ComponentRegistry {
     "Badge" story NodeData { Story of R.layout.fragment_badge },
     "Bar Chart" story NodeData { BarChartStory of R.layout.fragment_bar_chart },
     "Bottom Nav" story NodeData { BottomNavStory of R.layout.fragment_bottom_nav },
+    "Bottom Sheet" story NodeData { Story of R.layout.fragment_bottom_sheet scrollable false },
```

Comment:
`scrollable false` disables putting the component sample in scrollable container.